pmlc/dialect/pxa/transforms/stencil.cc

```diff
@@ -253,8 +302,209 @@ void Stencil::CollectStrideOneIndices() {
   }
 }
 
+// Test if idx in tensors[tensor_idx] is stride one index
+bool Stencil::IsStrideOne(mlir::BlockArgument idx, unsigned tensor_idx) {
+  return strideOne[tensor_idx].find(idx) != strideOne[tensor_idx].end();
+}
+
+// Test if idx in the tensors are stride one
+bool Stencil::ValidateStrideOne(mlir::BlockArgument idx, unsigned matrix_idx) {
+  switch (matrix_idx) {
+  case 0:
+    // Test if M is stride one for B(3) and C(2)
```

Comment:
This code seems correct, but the comment doesn't make sense to me.

Reply:
Fixed